types/argparse/index.d.ts

```diff
@@ -121,6 +121,10 @@ export interface ArgumentOptions {
     version?: string;
 }
 
+export class BooleanOptionalAction extends Action {
+    call(parser: ArgumentParser, namespace: Namespace, values: string | string[], optionString: string | null): void;
```

Comment:
`Action` defines `call` already.  Do you need to define it again in this subclass?

Reply:
If I leave it off it complains that call isn't defined, because `call` is abstract in `Action`.

Reply:
Ah good to know.  I wonder if TS has an affordance for pure type definitions... maybe not.